pandas/io/formats/style.py

```diff
@@ -178,6 +178,21 @@ def to_excel(self, excel_writer, sheet_name='Sheet1', na_rep='',
                         startcol=startcol, freeze_panes=freeze_panes,
                         engine=engine)
 
+    def to_html(self, filename, mode='w'):
```

Comment:
`filename` should be optional, and default to printing to stdout. (see how `DataFrame.to_html`) does it.

Reply:
And `filename` should be called `buf` to be more consistent with others.